### ISSUE TITLE
Making a log file to keep track of the simulation

### ISSUE THREAD

**Issue by pjve90** [pjve90/LCV_RD_ABM#22]
I have been stuck for too many days to be able to create a log file where to keep track when the simulation starts, when the population goes extinct, and when it ends for each parameter value.

When I do it without the log file, it works (by running the code in line 739), but when I try to record it in a log file it gives me error.

Do you have any idea how to make it work?

I will move forward with the calculations and plots, in the meantime

https://github.com/pjve90/LCV_RD_ABM/blob/97dbcc8b019b02363de07d748e7db811cf33cf17/Scenario%201/scenario_1_simulation.R#L1019

**Comment by dieterlukas** [comment]
I think I found a solution based on this website: https://www.datatechnotes.com/2017/09/logging-into-file-in-r-scripts.html. It looks like an extra function is needed to store other output into the log file. I have replaced lines 739 through 972 with an example that works on my computer (including storing the warning messages). Initially there is a setup of the file, it's location, the call to store warnings in the file, and the function to store other messages in the file with a timestamp, then whenever you display a message on the screen with 'cat', that same message is recorded with a timestamp in the log file, and finally the log file is closed and the warning messages redirected to the screen.
